code/src/index.css

```diff
@@ -1,13 +1,218 @@
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* movie page */
+.movieName {
+  display: flex;
+  position: relative;
+  transition: 0.6;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.movieImage {
+  opacity: 1;
+  border: 2px solid white;
+  margin-top: 15px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.movieName:hover .movieImage {
+  opacity: 0.3;
+  transition: opacity .3s ease-out;
+  -moz-transition: opacity .3s ease-out;
+  -webkit-transition: opacity .3s ease-out;
+  -o-transition: opacity .3s ease-out;
+}
+
+.movieName:hover .movieText {
+  opacity: 1;
+  transition: opacity .6s ease-out;
+  -moz-transition: opacity .6s ease-out;
+  -webkit-transition: opacity .6s ease-out;
+  -o-transition: opacity .6s ease-out;
+}
+
+.movieText {
+  position: absolute;
+  left: 20px;
+  bottom: 20px;
+  opacity: 0;
+  color: white;
+  transition: 0.6;
+  margin: 10px
+}
+
+.movieTitle {
+  margin-top: 0px;
+  margin-bottom: 10px;
+}
+
+.releaseDate {
+  margin-top: 0px;
+  margin-bottom: 0px;
+  font-size: 16px;
+  font-style: normal;
+}
+
+/* details page  */
+
+.backButton {
+  max-width: 35px;
+  filter: invert(100%);
+  margin-top: 2.5%;
+  margin-left: 2.5%;
+  transition: transform 0.6s;
+}
+
+.backButton:hover {
+  transform: scale(1.2);
+}
+
+.backButton:hover .backToMoviesText {
+  transform: scale(1.2);
+}
+
+.backToMoviesText {
+  margin-top: 2.5%;
+  margin-left: 10px;
+  margin-bottom: 0;
+  font-size: 150%;
+  font-weight: 500;
+}
+
+.goBack {
+  display: block;
+  align-items: center;
+  color: white;
+  text-decoration: none;
+  margin-bottom: 10%;
+}
+
+
+.backWrapper {
+  display: flex;
+  flex-direction: row;
+  transition: transform 0.6s;
+}
+
+.movieContainer {
+  display: grid;
+  grid-template-columns: 1fr;
+  margin-right: 20px;
+}
+
+.productPageContainer {
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 100%;
+  min-height: 100vh;
+}
+
+.detailImage {
+  border: 4px solid white;
+  width: 80%;
+}
+
+.titleRating {
+  display: flex;
+  flex-direction: row;
+}
+
+.title {
+  text-shadow: 1px 1px gray;
+}
+
+.detailsText {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  padding-left: 20px;
+  color: white;
+  justify-content: flex-end;
+}
+
+.rating {
+  background-color: white;
+  color: black;
+  padding: 1px;
+  margin-left: 10px;
+  font-size: 20px;
+}
+
+.overview {
+  font-size: 16px;
+}
+
+@media (max-width: 668px) {
+  .movieContainer {
+    display: grid;
+    grid-template-columns: 1fr;
+  }
+
+  .summary {
+    display: flex;
+    flex-direction: column;
+    min-height: 85vh;
+    align-items: center;
+  }
+}
+
+/* tablet */
+@media (min-width: 669px) and (max-width: 1023px) {
+  .movieContainer {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .summary {
+    display: flex;
+    flex-direction: column;
+    min-height: 85vh;
+    align-items: center;
+  }
+}
```

Comment:
We noticed that the poster get very big right before the screen reaches 1024px. Would be really nice if you put a max-width on it, or used some other way to limit the size.